Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.22 AS builder
+FROM golang:1.22.0 AS builder
```

Comment:
why use `.0` I suggest reverthing this or using `1.22.12`

Reply:
I was attempting to match this exactly to go.mod, to avoid go downloading another version of Go on build in docker